app/helpers/view_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ViewHelper
+  def govuk_link_to(body, url, external: false)
+    html_options = { class: 'govuk-link' }
```

Comment:
This means we can't pass in additional `html_options`, such as additional `class`es. Will this be too restrictive?

Reply:
Definitely a possibility! I'll seek to have `html_options` as a parameter.

Reply:
@tvararu I've updated the view helper now, see https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/242/commits/7ab5dde672b221fb20fdd29b40d87d28fcd42a9c. What do you think?